modules/motus/merge/main.nf

```diff
@@ -0,0 +1,46 @@
+VERSION = '3.0.1'
+
+process MOTUS_MERGE {
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::motus=3.0.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/motus:3.0.1--pyhdfd78af_0':
+        'quay.io/biocontainers/motus:3.0.1--pyhdfd78af_0' }"
+
+    input:
+    path input
+    path profile_version_yml, stageAs: 'profile_version.yml'
+    val dir_input
+    val biom_format
+
+    output:
+    path("*.txt") , optional: true, emit: txt
+    path("*.biom"), optional: true, emit: biom
+    path "versions.yml" , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = 'motus_merged'
+    def cmd_input = dir_input ? "-d ${input}" : "-i ${input.join(',')}"
```

Comment:
Is it possible to check the inputs by groovy?
```
def cmd_input = input.isDirectory() ? "-d ${input}" : "-i ${input.join(',')}"
```

Reply:
Ach I really need to remember about these file attribute things... good idea!